chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomInternalType.kt

```diff
@@ -0,0 +1,15 @@
+package com.pusher.chatkit.rooms.state
+
+import com.pusher.chatkit.CustomData
+
+internal data class JoinedRoomInternalType(
+    val id: String,
+    val name: String,
+    val isPrivate: Boolean,
+    val createdAt: Long,
+    val updatedAt: Long,
+    val deletedAt: Long?,
+    val pushNotificationTitleOverride: String?,
+    val customData: CustomData?,
+    val lastMessageAt: Long?
```

Comment:
How about the following importance order and deletion of `deletedAt` (for consistency with handling `room-deleted` event):
```kotlin
    val id: String,
    val name: String,
    val isPrivate: Boolean,
    val createdAt: Long,
    val updatedAt: Long,
    val lastMessageAt: Long?,
    val customData: CustomData?,
    val pushNotificationTitleOverride: String?
```

Reply:
👍 have updated